ci/ansible/all.yml

```diff
@@ -36,7 +36,8 @@ ovn_number_chassis: 5
 ########################
 network_start_cidr: "172.16.201.0/24"
 network_number: "5"
-ports_per_network: "50"
+ports_per_network: "500"
```

Comment:
You are right that we should make this configurable, but I wasn't quite sure how to do that quickly last night ...